### DISCUSSION TITLE
Add create post button in main UI

### DISCUSSION THREAD

**Discussion by Efreak** [aeharding/voyager#741] [Ideas]
I had to search for the button to create a post, it's off in a menu. Please add any of the following:

- a floating create post button when viewing a community (or when viewing a feed; the create page *does* allow selecting a community)
- a create button on the account page
- preferred: 4 settings for the second button at bottom, where the inbox currently is:
  - default/current: show inbox always, show a counter when there's unread messages
  - show inbox with counter when there's unread messages, otherwise show create post button. Inbox can be available in amount page to reread messages.
  - always show create post button. Inbox/notifications are only available from account page.
  - show both create button *and" inbox button (probably only available on large devices?)

**Comment by aeharding** [comment]
Thanks for the issue! I'm converting this to a discussion for now because this is more of a brainstorming thing (Voyager current implements it the way Apollo did)
